internal/repository/repository_default.go

```diff
@@ -48,3 +44,15 @@ func NewSectionsRepository(DB *sql.DB) *sec_repository.SectionRepository {
 		DB: DB,
 	}
 }
+
+func NewEmployeesRepository(DB *sql.DB) *employees_repository.EmployeesRepository {
+	return &employees_repository.EmployeesRepository{
+		DB: DB,
+	}
+}
+
+func NewInboundOrdersRepository(DB *sql.DB) *inboundOrders_repository.InboundOrdersRepository {
+	return &inboundOrders_repository.InboundOrdersRepository{
+		DB: DB,
+	}
+}
```

Comment:
Evita o export global do db se você usar o a função de inicializar os controllers

```suggestion
func NewEmployeesRepository(db *sql.DB) {
return employees_repository.NewEmployeesRepository(db)
}

func NewInboundOrdersRepository(db *sql.DB) return inboundOrders_repository.NewInboundOrdersRepository(db) 
}
```